### PR TITLE
chore(showcase): Add Century 21 Financial and Base Backpackers

### PR DESCRIPTION
**Pull request by shansmith01**
Just adding a couple of sites to the Gatsby showcase. Love your work team!

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -8381,6 +8381,29 @@
     - User Experience
   built_by: Binaria
   built_by_url: "https://binaria.com/"
+- title: Century 21 Financial
+  url: https://century21financial.co.nz/
+  main_url: https://century21financial.co.nz/
+  description: >
```
Comment (sidharthachatterjee) [inline docs/sites.yml]: Need to remove > and move the next line to this one

File: docs/sites.yml [hunk 2]
```diff
@@ -8381,6 +8381,29 @@
...
+    - Finance
+    - Business
+  built_by: Shannon Smith
+  built_by_url: https://www.powerboard.co.nz/clients
+  featured: false
+- title: Base Backpackers
+  url: https://www.stayatbase.com/
+  main_url: https://www.stayatbase.com/
+  description: >
+Base Backpackers is one of Australasia's biggest youth adventure tourism brands. They are super stoked to have one of the fastest websites in the tourism industry.
```
Comment (sidharthachatterjee) [inline docs/sites.yml]: Same with this

File: docs/sites.yml (hunk 1 above)
Comment (shansmith01) [reply docs/sites.yml]: Done

File: docs/sites.yml (hunk 2 above)
Comment (shansmith01) [reply docs/sites.yml]: Done as well :)

**Pull request merged**